addons/mail/models/mail_activity.py

```diff
@@ -482,6 +482,7 @@ def action_notify(self):
                 'mail.message_activity_assigned',
                 dict(
                     activity=activity,
+                    company=activity.env.company,
```

Comment:
Shouldn't it be sudo-ed like when base qweb use it ? Did you try in MC environments ?